Providers/Scripts/client.py

```diff
@@ -176,8 +186,15 @@ def main (argv):
         else:
             trace (ScriptsDir + '/3.x')
             os.chdir (ScriptsDir + '/3.x')
-        from Scripts import *
-        
+            sys.path.append(ScriptsDir + '/3.x/Scripts')
+        try:
+            from Scripts import *
+        except:
```

Comment:
This is added as a precautionary measure as the partners haven't already moved to python 3, some of their existing scripts may fail while import if they have syntax issues. This could lead to blocking other resources also to apply their configuration as all the scripts(across all providers) are in the same folder.